src/selection.coffee

```diff
@@ -404,6 +404,28 @@ class Selection extends Model
     @selectLeft() if @isEmpty() and not @editor.isFoldedAtScreenRow(@cursor.getScreenRow())
     @deleteSelectedText()
 
+  # Deprecated: Use {::deleteToBeginningOfWord} instead.
+  backspaceToBeginningOfWord: ->
```

Comment:
I believe because they're identical to their `deleteTo*` counterparts and to make the naming consistent? I can take it out though.